cmd/limactl/main.go

```diff
@@ -54,8 +54,13 @@ func newApp() *cobra.Command {
 		SilenceUsage:  true,
 		SilenceErrors: true,
 	}
+	rootCmd.PersistentFlags().Bool("silent", false, "silent mode")
```

Comment:
What about `--log-level  (debug|warn|info)`

Reply:
That would also work, as an alternative to `--debug` and `--silent` ?

Or maybe `--quiet` is a more common, but this one was _very_ quiet.

Reply:
I'm fine with `--log-level`, and I would prefer `--log-level warn` to be the default instead of `info`.

Reply:
Changing the default might be hard (and will result in losing the `message` in the YAML)

Reply:
> Changing the default might be hard

Why? Do you consider this an incompatible change, or would we actually be breaking something?

> (and will result in losing the message in the YAML)

No, it doesn't, just losing the "Message from the instance $INSTANCE" header, and we could write that directly to STDOUT too, if we want to:

```golang
	logrus.Infof("Message from the instance %q:", inst.Name)
	for scanner.Scan() {
		// Avoid prepending logrus "INFO" header, for ease of copypasting
		fmt.Println(scanner.Text())
	}
```

Reply:
I don't think we do (want to repeat the instance name again)

Reply:
Fine with me; I only included it to show how we can replicate the current behaviour:

```console
INFO[0090] READY. Run `limactl shell docker` to open the shell.
INFO[0090] Message from the instance "docker":
To run `docker` on the host (assumes docker-cli is installed), run the following commands:
------
docker context create lima-docker --docker "host=unix:///Users/jan/.lima/docker/sock/docker.sock"
docker context use lima-docker
docker run hello-world
------
```

We probably still want to have some kind of header line before the actual message; but I don't care too much how it looks like. Maybe

```console
INFO[0090] Message from the instance template:
```

Reply:
Just meant that it might not be needed.

```
? Creating an instance "default" Open an editor to review or modify the current configuration
Starting "default"

READY.
This will be shown to the user.

```

Reply:
> Just meant that it might not be needed.

Ok

> And now that we have the command wrappers, you don't even need to configure anything anyway.

We should still have a message that tells the user about the wrapper.

And I think it is still useful to tell them about the docker context option as well because if they use other tools to issue `docker` commands (VS Code plugins?), they will not be aware of the `docker.lima` wrapper, but might do the right thing when the context is set.

Reply:
In many unix utilities `silent` implies no output at all. Curl for instance wouldn't give you anything other than the output from the command.

Reply:
I guess `--quiet` would have been an option as well